### ISSUE TITLE
Times seem wrong

### ISSUE THREAD

**Issue by AmberSJones** [UCHIC/ODM2Sensor#80]
A lot of the times seems unfeasible- 3 am for a Site Visit, etc. I need to first check if these are correct (or not) in the database.

**Comment by marioscience** [comment]
Times were being off because of UTC Offset. UTC Offset was deactivated for the project since this is handled in the database.

**Comment by AmberSJones** [comment]
So, before, were the UTC times the ones being displayed? I don't see any of the 3 am times, but there are a few 11 pm. I still want to check these in the database.

**Comment by marioscience** [comment]
There are some instances on the databases of dates that are late at night. Here is an example: 
![image](https://cloud.githubusercontent.com/assets/5332522/8708555/e5af78c8-2af9-11e5-84ff-487b6858c07c.png). I tested inserting an site visit and checked the dates and everything looks correct.

**Comment by AmberSJones** [comment]
Ok. I am going to check these out and see if there is an issue in the database when I get a chance.

**Comment by AmberSJones** [comment]
So, I looked in the database to check out a bunch of times that seemed bizarre- i.e., late at night times, long duration of events (multiple days), and end times after start times. All of the ones that I identified were verified in the database- as well as in the original GAMUT database. This is probably a result of inadvertent time selection by the user. The technicians will need to resolve these going forward.
